src/psyclone/domain/lfric/lfric_kern.py

```diff
@@ -399,12 +412,7 @@ def colourmap(self):
             raise InternalError(f"Kernel '{self.name}' is not inside a "
                                 f"coloured loop.")
         sched = self.ancestor(InvokeSchedule)
-        if self._is_intergrid:
-            invoke = sched.invoke
-            if self._intergrid_ref is None:
-                raise InternalError(
-                    f"Colourmap information for kernel '{self.name}' has "
-                    f"not yet been initialised")
+        if self.is_intergrid:
```

Comment:
Please also update the docstring description of when this will raise an InternalError.

Reply:
Done